src/organization/screens/organization-repository-list.screen.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
```

Comment:
Is the idea that we'll need two separate screens (org-repo-list-screen and repo-list-screen) even if we've added normalized data throughout our store? Or is this just temporary?

Reply:
This was temporary but may come with a plot-twist :)

As you can see OrganizationRepositoryListScreen is now a simple container that calls a dumb `<RepositoryList />` component.

OrganizationRepositoryListScreen implements the store logic, and knows, being an organization screen, that it should read from the ORGS_GET_REPOS pagination.

For a user, we will be reading from USERS_GET_REPOS instead.

I guess it makes sense to have as many containers as "virtual" screens, and a single dumb component used by all of them. What do you think about it?